<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnrollPWD</class>
 <widget class="QDialog" name="EnrollPWD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblID">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPart">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>160</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Part</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblBackground">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblPwd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pwd</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtID">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitleIcon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>26</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitleBackground">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitleText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>240</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPwd">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK :Next             ESC:Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>98</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <zorder>lblPart</zorder>
  <zorder>lblBackground</zorder>
  <zorder>txtName</zorder>
  <zorder>lblPwd</zorder>
  <zorder>txtID</zorder>
  <zorder>lblTitleIcon</zorder>
  <zorder>lblTitleBackground</zorder>
  <zorder>lblTitleText</zorder>
  <zorder>txtPwd</zorder>
  <zorder>lblStatus</zorder>
  <zorder>pushButton</zorder>
  <zorder>lblName</zorder>
  <zorder>lblID</zorder>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>txtID</tabstop>
  <tabstop>txtPwd</tabstop>
  <tabstop>txtName</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
